collect_app/src/main/java/org/odk/collect/android/fragments/Camera2Fragment.java

```diff
@@ -82,15 +82,34 @@
     /**
      * Conversion from screen rotation to JPEG orientation.
      */
+
+    /*
+     * For front camera only.
+     * Bug fixed on 2018.2.17.
+     */
```

Comment:
I don’t think this comment adds much value, especially the note about the bug fix. We have issues, pull requests, and commit histories to record when bugs are fixed.